.vscode/launch.json

```diff
@@ -41,7 +41,7 @@
                 // metadata storage
                 "METADATA_USER_ID": "some-metadata-user-id"
```

Comment:
```suggestion
                "SYSTEM_USER_ID": "some-system-user-id"
```

Reply:
this PR is limited to the service rename, this can be done in a subsequent PR